sidebars.js

```diff
@@ -68,7 +68,11 @@ module.exports = {
         "parca-agent",
         "parca-agent-design",
         "parca-agent-labelling",
-        "parca-agent-language-support",
+        {
+          type: "category",
+          label: "Language Support",
```

Comment:
```suggestion
          label: "Languages",
```
Personally, I'd prefer a shorter menu name here.
Great idea with the languages!

Reply:
I agree with respect to having shorter menu names whenever possible. Although in this case, I think just 'Languages' can be a bit confusing and generic in the context of parca-agent.